### ISSUE TITLE
Why does it say "total step:0"?

### ISSUE THREAD

**Issue by g558800** [bmaltais/kohya_ss#2353]
00:34:11-982811 INFO     Total steps: 0
00:34:11-985926 INFO     Train batch size: 2
00:34:11-988929 INFO     Gradient accumulation steps: 1
00:34:11-992876 INFO     Epoch: 40
00:34:11-994870 INFO     max_train_steps (0 / 2 / 1 * 40 * 1) = 0
00:34:11-997880 INFO     stop_text_encoder_training = 0
00:34:12-000886 INFO     lr_warmup_steps = 0
00:34:12-006790 WARNING  Here is the trainer command as a reference. It will not be executed:

00:34:12-012353 INFO     "E:\AI\kohya_ss\venv\Scripts\accelerate.EXE" launch --dynamo_backend no --dynamo_mode default
                         --mixed_precision bf16 --num_processes 1 --num_machines 1 --num_cpu_threads_per_process 2
                         "E:/AI/kohya_ss/sd-scripts/train_network.py" --config_file "./outputs/tmpfilelora.toml"

00:34:12-017397 INFO     Showing toml config file: ./outputs/tmpfilelora.toml

00:34:12-017397 INFO     bucket_no_upscale = true
                         bucket_reso_steps = 64
                         cache_latents = true
                         cache_latents_to_disk = true
                         caption_dropout_rate = 0.01
                         caption_extension = ".txt"
                         clip_skip = 1
                         dynamo_backend = "no"
                         epoch = 40
                         full_bf16 = true
                         gradient_accumulation_steps = 1
                         huber_c = 0.1
                         huber_schedule = "snr"
                         learning_rate = 0.0001
                         loss_type = "l2"
                         lr_scheduler = "constant_with_warmup"
                         lr_scheduler_args = []
                         lr_scheduler_num_cycles = 2
                         lr_scheduler_power = 1.1
                         max_bucket_reso = 2048
                         max_data_loader_n_workers = 0
                         max_grad_norm = 1
                         max_timestep = 1000
                         max_token_length = 75
                         min_bucket_reso = 256
                         mixed_precision = "bf16"
                         network_alpha = 64
                         network_args = [ "conv_dim=16", "conv_alpha=16", "rank_dropout=0.1", "module_dropout=0.1",]
                         network_dim = 128
                         network_dropout = 0.1
                         network_module = "networks.lora"
                         noise_offset = 0.1
                         noise_offset_type = "Original"
                         optimizer_args = [ "weight_decay=0.001",]
                         optimizer_type = "AdamW"
                         output_dir = "D:/train_lora/train_output"
                         output_name = "liwanhua_woman"
                         pretrained_model_name_or_path =
                         "E:/AI/stable-diffusion-webui-forge/models/Stable-diffusion/awportrait_v13.safetensors"
                         prior_loss_weight = 1
                         resolution = "512,512"
                         sample_prompts = "D:/train_lora/train_output\\prompt.txt"
                         sample_sampler = "k_dpm_2_a"
                         save_every_n_epochs = 4
                         save_model_as = "safetensors"
                         save_precision = "bf16"
                         scale_weight_norms = 1
                         shuffle_caption = true
                         text_encoder_lr = 5e-5
                         train_batch_size = 2
                         train_data_dir = "D:/train_lora/train_input"
                         unet_lr = 0.0001
                         wandb_api_key = "False"
                         xformers = true

00:34:12-046866 INFO     end of toml config file: ./outputs/tmpfilelora.toml

It can't train.

**Comment by g558800** [comment]
![image](https://github.com/bmaltais/kohya_ss/assets/119348639/6bc05ba5-e121-4d33-aeea-1d527561e32a)

Addendum: There are pictures in the training folder.

**Comment by bmaltais** [comment]
In `D:/train_lora/train_input` do you see a folder that look something like `40_<wathever>`

**Comment by g558800** [comment]
yes.And this problem has been solved, it is the "image folder" in the "model" option box should be filled with the image directory after "data preparation".  Thanks for your interest in this issue.

**Comment by bmaltais** [comment]
Did you try pushing the Copy info to respective fields button? This is when it populate the fields:

![image](https://github.com/bmaltais/kohya_ss/assets/7474674/fa76dd5b-cc66-4fac-a666-29e7280057bf)

**Comment by g558800** [comment]
No ,i didn't.
